M2/Macaulay2/packages/ToricHigherDirectImages.m2

```diff
@@ -0,0 +1,1065 @@
+--- -*- coding: utf-8 -*-
+------------------------------------------------------------------------------
+-- Copyright 2025 Sasha Zotine
+--
+-- This program is free software: you can redistribute it and/or modify it
+-- under the terms of the GNU General Public License as published by the Free
+-- Software Foundation, either version 3 of the License, or (at your option)
+-- any later version.
+--
+-- This program is distributed in the hope that it will be useful, but WITHOUT
+-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+-- FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+-- more details.
+--
+-- You should have received a copy of the GNU General Public License along
+-- with this program.  If not, see <http://www.gnu.org/licenses/>.
+------------------------------------------------------------------------------
+newPackage(
+    "ToricHigherDirectImages",
+    Version => "1.0", 
+    Date => "2025 April",
+    Authors => {
+	{Name     => "Sasha Zotine", 
+	 Email    => "sashahbc@gmail.com"}
+	},
+    Headline => "computations involving pushforwards and higher direct images of toric maps",
+    DebuggingMode => false,
+    PackageExports => {"NormalToricVarieties","Complexes"},
+    PackageImports => {"FourierMotzkin"}
```

Comment:
Thanks for the contribution!

The test builds are failing because each package should have at least one keyword, ideally chosen from the list of headings in the [packages provided with Macaulay2](https://www.macaulay2.com/doc/Macaulay2/share/doc/Macaulay2/Macaulay2Doc/html/_packages_spprovided_spwith_sp__Macaulay2.html) documentation node.

These are given as an option to `newPackage`, e.g.,:

```m2
newPackage("
    "ToricHigherDirectImage",
    ...
    Keywords => {"Toric Geometry"},
    ...
```